prysm/run_beacon_node_peer.sh

```diff
@@ -30,4 +30,4 @@ fi
 run_beacon_node.sh \
     --min-sync-peers=1 \
     --bootstrap-node "$PEER" \
-    --interop-genesis-time "$INTEROP_GENESIS_TIME" $@
+    --interop-num-validators 4 $@
```

Comment:
I could be wrong, but shouldn't the main beacon node be the one generating validators? If only the peer does this then we cant run validators without the peer.